src/plot_splines.jl

```diff
@@ -3,22 +3,46 @@ using BSplineKit, Unfold
     plot_splines(m::UnfoldModel; kwargs...)
     plot_splines!(f::Union{GridPosition, GridLayout, Figure}, m::UnfoldModel; kwargs...)
 
-Shows two subfigures. First, baseses of splines. Second, density of splines.\\
-Dashed lines shows spline knots.
+Visualization of spline terms in an UnfoldModel. Two subplots are generated for each spline term:\\
+1) the basis function of the spline. 2) the density of the underlying covariate.\\
+
+Multiple spline terms are arranged across columns.
+Dashed lines indicate spline knots.
 
 ## Arguments:
 
 - `f::Union{GridPosition, GridLayout, Figure}`
     `Figure`, `GridLayout`, or `GridPosition` to draw the plot.
 - `m::UnfoldModel`\\
     UnfoldModel with splines.
+- `spline_axis::NamedTuple = (;)`\\
+    Here you can flexibly change configurations of spline subplots.\\
+    To see all options just type `?Axis` in REPL.
+- `density_axis::NamedTuple = (;)`\\
+    Here you can flexibly change configurations of density subplots.\\
+    To see all options just type `?Axis` in REPL.
+- `superlabel_axis::NamedTuple = (;)`\\
```

Comment:
is superlabel really an axis? or a plot handle?

Reply:
ah I think it is a "Block" - tricky concept, maybe superlabel_config ?

Reply:
I completely remade it
renamed to superlabel_config